pom.xml

```diff
@@ -86,22 +86,17 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>google-oauth-plugin</artifactId>
-      <version>0.6</version>
+      <version>0.9-SNAPSHOT</version>
```

Comment:
This needs to be changed to 0.9. SNAPSHOT is used for local testing/building while that version is unreleased.